net-im/skype-dbus-mock/skype-dbus-mock-0_p20181011.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6,3_7} )
+
+inherit python-r1
+
+EGIT_COMMIT="3a9e2882ac5c0ad6be3c5cb5c7da008b4cfa51da"
```

Comment:
how this variable name is relevant? It is from the `git-r3` eclass which is not being used

Reply:
What would be a more appropriate variable name here?